tests/pages/services/ServicePodReviewScreen-cy.js

```diff
@@ -1063,6 +1063,259 @@ describe("Services", function() {
       });
     });
 
+    it("renders proper review screen and JSON for a pod with two containers and ephemeral volume", function() {
+      const serviceName = "pod-with-ephemeral-volume";
+      const command = "`while true ; do echo 'test' ; sleep 100 ; done";
+      const cmdline = "while true; do echo 'test' ; sleep 100 ; done";
+
+      cy.contains("Multi-container (Pod)").click();
+
+      cy
+        .root()
+        .getFormGroupInputFor("Service ID *")
+        .type(`{selectall}{rightarrow}${serviceName}`);
+
+      cy.get(".menu-tabbed-item").contains("container-1").click();
+
+      // TODO: Due to a bug in cypress you cannot type values with dots
```

Comment:
We've upgraded cypress, shall we try uncommenting this part? ;)

Reply:
Then we should do this for all the tests.

Reply:
Let's add a follow up PR for that 👍

Reply:
C'mon! It's a new test!

Reply:
Let's please fix the new one and fix all the others in the followup

Reply:
New ©️ 🍝